index.html

```diff
@@ -64,71 +64,195 @@
 				padding:15px 0;
 				clear:both;
 			}
+
+			#builds table {
+				width:100%;
+				font-size: small;
+			}
+
+			#builds tr.even
+			{
+				background-color:#efefef;
+			}
+
+			#builds td {
+				vertical-align: top;
+				padding:0 10px 0 0;
+			}
+
+			#builds td.date, #builds td.commit {
+					white-space: nowrap;
+					width:1%;
+			}
+			#builds th {
+				text-align: left;
+			}
+
+			#builds th.branch
+			{
+				font-size: large;
+			}
 		</style>
 		<script type="text/javascript">
 			$( document ).ready(function() {
+				function keys(obj)
+				{
+						var keys = [];
+						for(var key in obj)
+						{
+								if(obj.hasOwnProperty(key))
+								{
+										keys.push(key);
+								}
+						}
+						return keys;
+				}
+
+				function format_size(bytes) {
+					var i = 0;
+					var units = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
By the way, Nautilus on Linux (GNOME 3) seems to use KB = 10^3 bytes rather than K(i)B = 2^10 B.